api/comments.rb

```diff
@@ -62,5 +62,4 @@
       pass
     end
   end
-  comment.to_hash.to_json
```

Comment:
I see.  I'm getting the following error:
```
1) Comment API DELETE /api/v1/comments/:comment_id can delete a sub comment
     Failure/Error: delete "/api/v1/comments/#{child_comment.id}"
     RuntimeError:
       can't modify frozen BSON::Document
     # ./models/comment.rb:117:in `update_cached_child_count'
     # ./models/comment.rb:111:in `get_cached_child_count'
     # ./models/comment.rb:106:in `to_hash'
     # ./api/comments.rb:65:in `block in <top (required)>'
     # ./spec/api/comment_spec.rb:219:in `block (3 levels) in <top (required)>'
```
Either I'll eat the error, or determine what this really should be returning to not get a top-level error.  This may require an edx-platform change though.